docker-compose.yaml

```diff
@@ -177,7 +198,7 @@ services:
       - BRAVE_API_KEY=${BRAVE_API_KEY}
       - CARBON_AI_API_KEY=${CARBON_AI_API_KEY}
     entrypoint: /bin/bash
-    command: -c "set -e; prisma db push --schema ./backend/database/schema.prisma && uvicorn --host 0.0.0.0 --port 8000 backend.server.app:app --reload"
+    command: -c "set -e; prisma db push --accept-data-loss --schema ./backend/database/schema.prisma && uvicorn --host 0.0.0.0 --port 8000 backend.server.app:app --reload"
```

Comment:
Why would we want to accept data loss?
I don't think this should be the case always

Reply:
In my opinion, we should stop using `db push` and rather use `prisma migrate` and follow proper migrations process

Reply:
this was used when I a another branch had a different schema, we'll remove that for now, will do the migration PR later.